packages/cli/lang/en.lyaml

```diff
@@ -529,6 +529,28 @@ en:
             projectAlreadyExists: "A project with name {{ projectName }} already exists. Please choose another name."
             invalidAppId: "[--appId] Could not find appId {{ appId }}. Please choose another public app."
             invalidMarketplaceApp: "[--appId]: Could not migrate appId {{ appId }}. Marketplace apps cannot be migrated at this time. Please choose another public app."
+        cloneApp:
+          describe: "Clone a public app using the projects framework"
+          examples:
+            default: "Clone a public app using the projects framework"
+          options:
+            appId:
+              describe: "The ID for the public app being cloned"
+            location:
+              describe: "Directory where project should be created"
+          enterLocation: "[--location] Where should the cloned project be created?"
+          cloneStatus:
+            inProgress: "Cloning app configuration to {{#bold}}public-app.json{{/bold}} component definition ..."
+            done: "Cloning app configuration to public-app.json component definition ... DONE"
+            success: "Your cloned project was created in {{ location }}"
+            failure: "Cloning app configuration to public-app.json component definition ... FAILED"
+          errors:
+            invalidAccountTypeTitle: "{{#bold}}Developer account not targeted{{/bold}}"
+            invalidAccountTypeDescription: "Only public apps created in a developer account can be converted to a project component. Select a connected developer account with {{useCommand}} or {{authCommand}} and try again."
+            invalidAppId: "[--appId] Could not find appId {{ appId }}. Please choose another public app."
```

Comment:
No need for the mention of the `[--appId]` flag here. We typically only do this for prompt options

Reply:
I've addressed that issue in this [PR](https://github.com/HubSpot/hubspot-cli/pull/1111). Going to leave as is to prevent merge conflicts.

Reply:
Merged in these changes here: https://github.com/HubSpot/hubspot-cli/pull/1085/commits/9d7e99a5965e912bdad29344fc2265453c60f7a7